content/app/_index.en.md

```diff
@@ -0,0 +1,15 @@
+---
+title: App
+weight: 1
+pre: "<b>1. </b>"
+chapter: true
+description: "Shiny APP"
+---
+
+### Section 3
+
+# Shiny APP examples
+
+This section will contain examples creating APP with demographics output.
```

Comment:
maybe after APP add: (interactive data display) or something like that for people completely new to what this is